cmd/coder/main.go

```diff
@@ -15,31 +15,35 @@ import (
 	"go.coder.com/flog"
 )
 
-var (
-	version string = "unknown"
-)
+// Using a global for the version so it can be set at build time using ldflags.
+var version string = "unknown"
 
 func main() {
 	ctx, cancel := context.WithCancel(context.Background())
 	defer cancel()
 
+	// If requested, spin up the pprof webserver.
 	if os.Getenv("PPROF") != "" {
 		go func() {
```

Comment:
I wish we put panic handlers in all go routines.

Reply:
Hmm, panics are developer errors, not really expected in production. As a safety net on a service to log/alert about a bug makes sense, and in general, for anything expected to be long-running, I do agree, we should have panic handler, but here, we are in a CLI and specifically in a debug handler, so it is probably not necessary.